readthedocs/builds/migrations/0007_add-automation-rules.py

```diff
@@ -25,7 +25,7 @@ class Migration(migrations.Migration):
                 ('priority', models.IntegerField(help_text='A lower number (0) means a higher priority', verbose_name='Rule priority')),
                 ('match_arg', models.CharField(help_text='Value used for the rule to match the version', max_length=255, verbose_name='Match argument')),
                 ('action', models.CharField(choices=[('activate-version', 'Activate version on match'), ('set-default-version', 'Set as default version on match')], max_length=32, verbose_name='Action')),
-                ('action_arg', models.CharField(blank=True, help_text='Value used for the action to perfom an operation', max_length=255, null=True, verbose_name='Action argument')),
+                ('action_arg', models.CharField(blank=True, help_text='Value used for the action to perform an operation', max_length=255, null=True, verbose_name='Action argument')),
```

Comment:
Existing migrations shouldn't be changed, we need to create a new one with the typo fixed.

Reply:
Will revert.